src/app/components/Transcript/index.styles.tsx

```diff
@@ -0,0 +1,121 @@
+import { css, Theme } from '@emotion/react';
+import pixelsToRem from '#app/utilities/pixelsToRem';
+import { focusIndicatorThickness } from '../ThemeProvider/focusIndicator';
+
+export default {
+  arrowShape: ({ palette, isDarkUi }: Theme) =>
```

Comment:
FYI this looks like a rectangle when you change colours in Firefox... is there anything we can do to improve the affordance with CSS, should we switch back to the SVG?

Reply:
Yh I think this makes sense. The CSS shape code is more complicated.

Reply:
I think I will just change back to SVG to save time.

It could be possible to achieve just with CSS with more work/ changing the shape to be something more easily achieveable with just borders e.g. [here](https://bbc.github.io/accessibility-news-and-you/guides/screen-reader-ux.html). The issue is that I've achieved this shape with transparent borders and background colours. But this could be a future improvement